modules/message/html/compose.js

```diff
@@ -53,7 +53,26 @@ exports.create = function (api) {
       placeholder
     })
 
+    var warningMessage = Value(null)
+    var warning = h('section.warning',
+      { className: when(warningMessage, '-open', '-closed') },
+      [
+        h('div.warning', warningMessage),
+        h('div.close', { 'ev-click': () => warningMessage.set(null) }, 'x')
+      ]
+    )
     var fileInput = api.blob.html.input(file => {
+      const megabytes = file.size / 1024 / 1024
+      if (megabytes >= 5) {
+        const rounded = Math.floor(megabytes*100)/100
```

Comment:
i'm confused what this does. does this round a decimal number to two digits?

Reply:
yep, is there a tidier way ?